ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh

```diff
@@ -258,6 +258,23 @@ retry_with_timeout 400 5 kubectl rollout status daemonset linuxptp-daemon -nopen
 # Run ptp conformance test
 cd -
 echo "running conformance tests from branch ${TEST_BRANCH}"
+
+# Set go version
+if [[ "$T5CI_VERSION" =~ 4.1[2-5]+ ]]; then # 4.12 to 4.15
+    PATH_VAL=$(echo $PATH | sed -e 's#:/usr/local/1.20/go/bin:/go/bin##g')
+    export PATH=$PATH_VAL
+elif [[ "$T5CI_VERSION" == "4.16" ]]; then # 4.16
+    PATH_VAL=$(echo $PATH | sed -e 's#:/usr/local/1.21.11/go/bin:/go/bin##g')
+    export PATH=$PATH_VAL
+else # 4.17 and 4.18
+    PATH_VAL=$(echo $PATH | sed -e 's#:/usr/local/1.22.4/go/bin:/go/bin##g')
+    export PATH=$PATH_VAL
+fi
+
+#all versions run latest go for test build.
+source $HOME/golang-1.22.4
```

Comment:
This has no effect in running the testw since `hack/run-functests.sh` script will be running in its own shell. 
This will cause problem in `make undeploy' in line 395 for older release.

Reply:
will close this one out